src/Components/dashboard/route/AddRouteInit.jsx

```diff
@@ -8,148 +18,175 @@ import { Link } from "react-router-dom";
 import AddIcon from "@mui/icons-material/Add";
 import List from "@mui/material/List";
 
-import React, { useState } from "react";
+
 import TextField from "@mui/material/TextField";
 
 import Buttons from "../operatorsanddriver/Button.jsx";
 import "./AddRoutes.css";
 import ListItem from "@mui/material/ListItem";
+import { saveRoute } from "../../../redux/Action/routes";
+import { useEffect } from "react";
 
 const un = "Bus stop one, You can add others by hitting + button";
 
 const AddRouteInit = () => {
-  const [routeID, setRouteID] = useState("");
-  const [startPoint, setStartPoint] = useState("");
-  const [endPoint, setEndPoint] = useState("");
-  const [stop, setStop] = useState("");
-
-  const [routeIDError, setRouteIDError] = useState(false);
-  const [startPointError, setStartPointError] = useState(false);
-  const [endPointError, setEndPointError] = useState(false);
-  const [stopError, setStopError] = useState(false);
-
-  const handleClear = () => {
-    console.log("Content cleared");
-  };
-
-  const handleAddRoute = () => {
-    console.log("route added");
 
-    const List = document.querySelector(".list-route");
-    const ListItem = document.createElement("li");
-    ListItem.style.border = "1px solid #c4c4c4";
-    ListItem.style.width = "70%";
-    ListItem.style.height = "40px";
-    ListItem.style.paddingLeft = "20px";
+    const [routeCode, setRouteID] = useState("");
+    const [startLocation, setStartPoint] = useState("");
+    const [endLocation, setEndPoint] = useState("");
+    const [stop, setStop] = useState("");
+    const [name, setName] = useState("");
+    const [distance, setDistance] = useState("");
+    const [duration, steDuration] = useState("");
+    const [success, setSuccessMessage] =useState(false);
+    const [errorMessage,setErrorMessage] = useState('')
+    const [address, setAddress] =useState("");
+    const [coordinates, setCoordinates] =useState({lat:null , lng:null});
+
+
+    const [routeIDError, setRouteIDError] = useState(false);
+    const [startPointError, setStartPointError] = useState(false);
+    const [endPointError, setEndPointError] = useState(false);
+    const [stopError, setStopError] = useState(false);
+
+    const routeSave = useSelector((state) => state.saveRoute);
+    const { loading, routes, error } = routeSave;
+
+    const dispatch = useDispatch();
+
+    const handleClear = () => {
+        console.log("Content cleared");
```

Comment:
Use this function `const handleClear = () => {
        console.log("Content cleared");`